### DISCUSSION TITLE
blog/convert-threejs-to-angular-three/

### DISCUSSION THREAD

**Discussion by giscus** [nartc/tnc-scully#53] [Announcements]
# blog/convert-threejs-to-angular-three/

Personal blog by Chau Tran

https://nartc.me/blog/convert-threejs-to-angular-three/

**Comment by comfortme** [comment]
a secret angular-three version: "^2.0.0-beta.226" 😱 When are we going to see the stable version of Angular Three 2.0.0 🙏

**Comment by omidkh68** [comment]
How can I create a 3D object which created with Three.js that I want to render it into the mapbox in my angular project?
